quic/client/handshake/ClientHandshake.cpp

```diff
@@ -15,8 +15,8 @@
 
 namespace quic {
 
-ClientHandshake::ClientHandshake(QuicCryptoState& cryptoState)
-    : cryptoState_(cryptoState), visitor_(*this) {}
+ClientHandshake::ClientHandshake(QuicCryptoState &cryptoState)
```

Comment:
We generally cuddle & with the type instead of the variable name. Here and elsewhere, can you change it so that & is next to the type?

Reply:
My bad. Could you prioritize #43 so that this kind of problem doesn't happen as often, please ?